Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
```

Comment:
Jeg tror måske også det er bedre at bygge den fra 14:04, da det væsentlige i processen er at vi henter 14:04 som PhantomJS passer til. Node-versionen er knapt så relevant så vidt jeg kan se.

Der er jo ikke nogen garanti for at dockerfile/nodejs bliver ved med at være 14:04.